bees/discordbee/discordbee.go

```diff
@@ -130,11 +130,26 @@ func (mod *DiscordBee) Action(action bees.Action) []bees.Placeholder {
 		if err != nil {
 			mod.LogErrorf("Unable to send message: %v", err)
 		}
+	case "send_news":
+		var contents string
+		var channelID string
+		var message *discordgo.Message
+		action.Options.Bind("contents", &contents)
+		action.Options.Bind("channel_id", &channelID)
+
+		message, err := mod.discord.ChannelMessageSend(channelID, contents)
+		if err != nil {
+			mod.LogErrorf("Unable to send message: %v", err)
+		}
+        message, err2 := mod.discord.ChannelMessageCrosspost(channelID, message.ID)
```

Comment:
This looks like an indentation error. Make sure to format your sources with `gofmt` (part of the Go toolchain), a feature you can typically enable in your IDE. This will automatically take care of formatting your sources and fixes such issues.

Reply:
Oh my, how this is possible as I have compiled this exact code and it worked? 😱 
Sorry, I'll check that quickly (first time I try go 😓 )

Reply:
It's not a strict error and will still compile (unlike Python where indentation is part of the syntax). Obviously it still harms readability tho, so it's something we should fix :smiley:

Let me know if I can help you getting started with Go!

Reply:
> Let me know if I can help you getting started with Go!
Thank you for the offer! Beauty of Free Software, I needed the function so I add it and give it back to the project!
But still I will need to read some basics Go documentation before doing something more elaborated ^^

I have fixed the indentation error.
In fact, I used 4 spaces instead of tabs, and my IDE show tabs as 4 spaces... So visually it was perfect on my IDE, problem appears only once git pushed 😅

Now all files should be fixed 👍